algorithms/src/Kokkos_Sort.hpp

```diff
@@ -628,10 +613,23 @@ std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
   bin_sort.sort(exec, view);
 }
 
+template <class ExecutionSpace, class DataType, class... Properties>
+std::enable_if_t<(Kokkos::is_execution_space<ExecutionSpace>::value) &&
+                 (SpaceAccessibility<
+                     HostSpace, typename Kokkos::View<DataType, Properties...>::
+                                    memory_space>::accessible)>
+sort(const ExecutionSpace& space,
+     const Kokkos::View<DataType, Properties...>& view) {
+  space.fence("Kokkos::sort: Fence before sorting on the host");
+  auto first = Experimental::begin(view);
+  auto last  = Experimental::end(view);
+  std::sort(first, last);
```

Comment:
why are we not using a std policy for this?

Reply:
@jstrzebonski @dalg24 last comment before approving, but why not using a policy here?

Reply:
Because say we are using the `Serial` backend, we are not allowed to just go grab more resources for sorting.
Say the execution space is any of our host parallel backend, we would have to honor what is available in the execution space instance.

Reply:
makes sense